seshcli/root.go

```diff
@@ -3,24 +3,29 @@ package seshcli
 import (
 	"fmt"
 
+	"github.com/joshmedeski/sesh/v2/git"
+	"github.com/joshmedeski/sesh/v2/home"
 	"github.com/joshmedeski/sesh/v2/lister"
-	"github.com/joshmedeski/sesh/v2/namer"
 	cli "github.com/urfave/cli/v2"
 )
 
-func Root(l lister.Lister, n namer.Namer) *cli.Command {
```

Comment:
With my request to move the logic of determining the name in the namer package, we can reverse this change.

Reply:
I did move the name parsing logic into the namer as you requested 👍, but we need to think on how we want to wire it up to [here]. Do we just import the namer, or dependency inject it? Maybe we can pair on this.